tt_metal/hw/inc/dataflow_api.h

```diff
@@ -415,6 +415,14 @@ uint64_t get_l1_noc_addr(const uint32_t id, const uint32_t page_size, const uint
     return noc_addr;
 }
 
+uint64_t get_l1_noc_addr_single_bank(uint32_t addr){
```

Comment:
how is this being used?

Reply:
ah its not used anymore, I was using it before and forgot to remove it.